diff --git a/src/modules/partition/Config.cpp b/src/modules/partition/Config.cpp
index e3c9f3b..2315ac2 100644
--- a/src/modules/partition/Config.cpp
+++ b/src/modules/partition/Config.cpp
@@ -460,6 +460,8 @@ Config::setConfigurationMap( const QVariantMap& configurationMap )
     Calamares::GlobalStorage* gs = Calamares::JobQueue::instance()->globalStorage();
     gs->insert( "armInstall", Calamares::getBool( configurationMap, "armInstall", false ) );
     fillGSConfigurationEFI( gs, configurationMap );
+    gs->insert( "createHybridBootloaderLayout", Calamares::getBool( configurationMap, "createHybridBootloaderLayout", false ) );
+    gs->insert( "enableLuksAutomatedPartitioning", Calamares::getBool( configurationMap, "enableLuksAutomatedPartitioning", false ) );
     fillConfigurationFSTypes( configurationMap );
 }
