diff --git a/src/modules/keyboard/Config.cpp b/src/modules/keyboard/Config.cpp
index a7c8160..ed01e3c 100644
--- a/src/modules/keyboard/Config.cpp
+++ b/src/modules/keyboard/Config.cpp
@@ -245,8 +245,8 @@ applyXkb( const BasicLayoutInfo& settings, AdditionalLayoutInfo& extra )
         }
 
         basicArguments.append(
-            xkbmap_layout_args_with_group_switch( { extra.additionalLayout, settings.selectedLayout },
-                                                  { extra.additionalVariant, settings.selectedVariant },
+            xkbmap_layout_args_with_group_switch( { settings.selectedLayout, extra.additionalLayout },
+                                                  { settings.selectedVariant, extra.additionalVariant },
                                                   extra.groupSwitcher ) );
         QProcess::execute( "setxkbmap", basicArguments );
 
@@ -271,8 +271,8 @@ applyLocale1( const BasicLayoutInfo& settings, AdditionalLayoutInfo& extra )
 
     if ( !extra.additionalLayout.isEmpty() )
     {
-        layout = extra.additionalLayout + "," + layout;
-        variant = extra.additionalVariant + "," + variant;
+        layout = layout + "," + extra.additionalLayout;
+        variant = variant + "," + extra.additionalVariant;
         option = extra.groupSwitcher;
     }
 
diff --git a/src/modules/keyboard/SetKeyboardLayoutJob.cpp b/src/modules/keyboard/SetKeyboardLayoutJob.cpp
index 81eba13..3131f1b 100644
--- a/src/modules/keyboard/SetKeyboardLayoutJob.cpp
+++ b/src/modules/keyboard/SetKeyboardLayoutJob.cpp
@@ -279,8 +279,8 @@ SetKeyboardLayoutJob::writeX11Data( const QString& keyboardConfPath ) const
               "        MatchIsKeyboard \"on\"\n";
 
 
-    const QStringList layouts = removeEmpty( { m_additionalLayoutInfo.additionalLayout, m_layout } );
-    const QStringList variants = removeEmpty( { m_additionalLayoutInfo.additionalVariant, m_variant } );
+    const QStringList layouts = removeEmpty( { m_layout, m_additionalLayoutInfo.additionalLayout } );
+    const QStringList variants = removeEmpty( { m_variant, m_additionalLayoutInfo.additionalVariant } );
     stream << "        Option \"XkbLayout\" \"" << layouts.join( "," ) << "\"\n";
     stream << "        Option \"XkbVariant\" \"" << variants.join( "," ) << "\"\n";
     if ( !m_additionalLayoutInfo.additionalLayout.isEmpty() )
@@ -313,8 +313,8 @@ SetKeyboardLayoutJob::writeDefaultKeyboardData( const QString& defaultKeyboardPa
     }
     QTextStream stream( &file );
 
-    const QStringList layouts = removeEmpty( { m_additionalLayoutInfo.additionalLayout, m_layout } );
-    const QStringList variants = removeEmpty( { m_additionalLayoutInfo.additionalVariant, m_variant } );
+    const QStringList layouts = removeEmpty( { m_layout, m_additionalLayoutInfo.additionalLayout } );
+    const QStringList variants = removeEmpty( { m_variant, m_additionalLayoutInfo.additionalVariant } );
     stream << "# KEYBOARD CONFIGURATION FILE\n\n"
               "# Consult the keyboard(5) manual page.\n\n";
 
