diff -ruN '--exclude=.git' compiz-aur-patched/compizconfig/ccsm/ccm/Pages.py compiz-final/compizconfig/ccsm/ccm/Pages.py
--- compiz-aur-patched/compizconfig/ccsm/ccm/Pages.py	2026-07-14 21:54:15.485931231 +0200
+++ compiz-final/compizconfig/ccsm/ccm/Pages.py	2026-07-14 21:54:35.706508379 +0200
@@ -670,9 +670,9 @@
             self.ProfileComboBox.append_text(profile.Name)
             if name == self.Context.CurrentProfile.Name:
                 active = i
+        self.ProfileComboBox.set_active(active+1)
         self.ProfileHandler = self.ProfileComboBox.connect("changed",
             self.ProfileChangedAddTimeout)
-        self.ProfileComboBox.set_active(active+1)
         profileAdd.connect("clicked", self.AddProfile)
         profileRemove.connect("clicked", self.RemoveProfile)
         profileBox.pack_start(self.ProfileComboBox, True, True, 0)
diff -ruN '--exclude=.git' compiz-aur-patched/compizconfig/libcompizconfig/backend/src/ini.c compiz-final/compizconfig/libcompizconfig/backend/src/ini.c
--- compiz-aur-patched/compizconfig/libcompizconfig/backend/src/ini.c	2026-07-14 21:54:15.494931607 +0200
+++ compiz-final/compizconfig/libcompizconfig/backend/src/ini.c	2026-07-14 21:54:35.706426404 +0200
@@ -597,7 +597,7 @@
 {
     int length = strlen (name->d_name);
 
-    if (strncmp (name->d_name + length - 6, ".ini++", 6))
+    if (strncmp (name->d_name + length - 6, ".ini", 6))
 	return 0;
 
     return 1;
diff -ruN '--exclude=.git' compiz-aur-patched/metadata/core.xml.in compiz-final/metadata/core.xml.in
--- compiz-aur-patched/metadata/core.xml.in	2026-07-14 21:54:15.609724128 +0200
+++ compiz-final/metadata/core.xml.in	2026-07-15 00:51:14.907772178 +0200
@@ -8,7 +8,23 @@
 		<_short>Active Plugins</_short>
 		<_long>List of currently active plugins</_long>
 		<type>string</type>
-		<default_plugins/>
+<default>
+		    <value>composite</value>
+		    <value>opengl</value>
+		    <value>decor</value>
+		    <value>grid</value>
+		    <value>move</value>
+		    <value>place</value>
+		    <value>regex</value>
+		    <value>resize</value>
+		    <value>snap</value>
+		    <value>text</value>
+		    <value>compiztoolbox</value>
+		    <value>ring</value>
+		    <value>wall</value>
+		    <value>animation</value>
+		    <value>blur</value>
+		</default>
  	    </option>
  	    <option name="audible_bell" type="bool">
 		<_short>Audible Bell</_short>
@@ -254,7 +269,7 @@
 			<option name="hsize" type="int">
 			<_short>Horizontal Virtual Size</_short>
 			<_long>Screen size multiplier for horizontal virtual size</_long>
-			<default>4</default>
+			<default>1</default>
 			<min>1</min>
 			<max>32</max>
 			</option>
diff -ruN '--exclude=.git' compiz-aur-patched/plugins/animation/animation.xml.in compiz-final/plugins/animation/animation.xml.in
--- compiz-aur-patched/plugins/animation/animation.xml.in	2026-07-14 21:54:15.559723532 +0200
+++ compiz-final/plugins/animation/animation.xml.in	2026-07-14 21:54:47.968180423 +0200
@@ -168,7 +168,7 @@
 	    <extensible/>
 	    <sort start="2"/>
 	    <default>
-	      <value>animation:Magic Lamp</value>
+	      <value>animation:Glide 2</value>
 	    </default>
 	  </option>
 	  <option name="minimize_durations" type="list">
@@ -226,7 +226,7 @@
             <extensible/>
             <sort start="2"/>
             <default>
-              <value>animation:Magic Lamp</value>
+              <value>animation:Glide 2</value>
             </default>
           </option>
           <option name="unminimize_durations" type="list">
diff -ruN '--exclude=.git' compiz-aur-patched/plugins/blur/blur.xml.in compiz-final/plugins/blur/blur.xml.in
--- compiz-aur-patched/plugins/blur/blur.xml.in	2026-07-14 21:54:15.502905907 +0200
+++ compiz-final/plugins/blur/blur.xml.in	2026-07-14 21:54:35.706174359 +0200
@@ -42,7 +42,7 @@
 	    <option name="alpha_blur_match" type="match">
 		<_short>Alpha blur windows</_short>
 		<_long>Windows that should be use alpha blur by default</_long>
-		<default></default>
+		<default>class=Xfce4-panel | class=Wrapper-2.0</default>
 	    </option>
 	    <option name="alpha_blur" type="bool">
 		<_short>Alpha Blur</_short>
diff -ruN '--exclude=.git' compiz-aur-patched/plugins/composite/composite.xml.in compiz-final/plugins/composite/composite.xml.in
--- compiz-aur-patched/plugins/composite/composite.xml.in	2026-07-14 21:54:15.504039497 +0200
+++ compiz-final/plugins/composite/composite.xml.in	2026-07-14 21:54:35.706098616 +0200
@@ -24,15 +24,15 @@
	    <option name="unredirect_fullscreen_windows" type="bool">
		<_short>Unredirect Fullscreen Windows</_short>
		<_long>Allow drawing of fullscreen windows to not be redirected to offscreen pixmaps</_long>
-		<default>true</default>
+		<default>false</default>
	    </option>
	    <option name="unredirect_match" type="match">
		<_short>Unredirect Match</_short>
		<_long>Specifies which windows will be unredirected when they are fullscreen. You might want to exclude video players for example, to avoid tearing. But this will be at the expense of performance/frame rate. Note: "class=" matching requires the regex plugin to work.</_long>
-		<default>(any) &amp; !(class=Totem) &amp; !(class=MPlayer) &amp; !(class=vlc) &amp; !(class=Plugin-container) &amp; !(class=QtQmlViewer) &amp; !(class=Firefox) &amp; !(class=google-chrome) &amp; !(class=google-chrome-unstable) &amp; !(class=chromium-browser)</default>
+		<default>(any)</default>
	    </option>
	    <option name="force_independent_output_painting" type="bool">
		<_short>Force independent output painting.</_short>
		<_long>Paint each output device independly, even if the output devices overlap</_long>
-		<default>false</default>
+		<default>true</default>
	    </option>
	</options>
    </plugin>
diff -ruN '--exclude=.git' compiz-aur-patched/plugins/grid/grid.xml.in compiz-final/plugins/grid/grid.xml.in
--- compiz-aur-patched/plugins/grid/grid.xml.in	2026-07-14 21:54:15.583723818 +0200
+++ compiz-final/plugins/grid/grid.xml.in	2026-07-14 21:54:35.706288574 +0200
@@ -578,20 +578,20 @@
 			<_short>Preview Outline Color</_short>
 			<_long>Color and opacity of the resize indicator preview outline.</_long>
 			<default>
-			    <red>0xfbfb</red>
-			    <green>0x8b8b</green>
-			    <blue>0x0</blue>
-			    <alpha>0x9f9f</alpha>
+			    <red>0x3f3f</red>
+			    <green>0xb8b8</green>
+			    <blue>0xbfbf</blue>
+			    <alpha>0x7f7f</alpha>
 			</default>
 		</option>
 		<option name="fill_color" type="color">
 			<_short>Preview Fill Color</_short>
 			<_long>Fill color and opacity of the resize indicator preview.</_long>
 			<default>
-			    <red>0xfbfb</red>
-			    <green>0x8b8b</green>
-			    <blue>0x0</blue>
-			    <alpha>0x4f4f</alpha>
+			    <red>0x1818</red>
+			    <green>0x8787</green>
+			    <blue>0x7c7c</blue>
+			    <alpha>0x7f7f</alpha>
 			</default>
 		</option>
 		<option name="disable_blend" type="bool">
diff -ruN '--exclude=.git' compiz-aur-patched/plugins/opengl/opengl.xml.in compiz-final/plugins/opengl/opengl.xml.in
--- compiz-aur-patched/plugins/opengl/opengl.xml.in	2026-07-14 21:54:15.511111834 +0200
+++ compiz-final/plugins/opengl/opengl.xml.in	2026-07-14 21:54:35.706023524 +0200
@@ -34,7 +34,7 @@
 	    <option name="sync_to_vblank" type="bool">
 		<_short>Sync To VBlank</_short>
 		<_long>Only perform screen updates during vertical blanking period</_long>
-		<default>true</default>
+		<default>false</default>
 	    </option>
 	    <option name="texture_compression" type="bool">
 		<_short>Texture Compression</_short>
@@ -58,7 +58,7 @@
 	    <option name="unredirect_driver_blacklist" type="string">
 		<_short>Unredirect Driver Blacklist</_short>
 		<_long>If non-empty, specifies a POSIX (extended) regular expression to match against the OpenGL driver strings (newline separated): "GL_VENDOR\nGL_RENDERER\nGL_VERSION". If the regular expression matches a substring of that concatenation then no windows will ever be unredirected while using that particular graphics driver.</_long>
-		<default>(nouveau|Intel).*Mesa 8.0</default>
+		<default></default>
 	    </option>
 
 	    <group>
diff -ruN '--exclude=.git' compiz-aur-patched/plugins/place/place.xml.in compiz-final/plugins/place/place.xml.in
--- compiz-aur-patched/plugins/place/place.xml.in	2026-07-14 21:54:15.512079320 +0200
+++ compiz-final/plugins/place/place.xml.in	2026-07-14 21:54:35.706380728 +0200
@@ -84,21 +84,33 @@
 			<_short>Positioned windows</_short>
 			<_long>Windows that should be positioned by default</_long>
 			<type>match</type>
+			<default>
+			    <value>(name=xfce4-popup-whiskermenu) | (class=Wrapper-2.0)</value>
+			</default>
 		    </option>
 		    <option name="position_x_values" type="list">
 			<_short>X Positions</_short>
 			<_long>X position values</_long>
 			<type>int</type>
+			<default>
+			    <value>-32768</value>
+			</default>
 		    </option>
 		    <option name="position_y_values" type="list">
 			<_short>Y Positions</_short>
 			<_long>Y position values</_long>
 			<type>int</type>
+			<default>
+			    <value>32767</value>
+			</default>
 		    </option>
 		    <option name="position_constrain_workarea" type="list">
 			<_short>Keep In Workarea</_short>
 			<_long>Keep placed window in work area, even if that means that the position might differ from the specified position</_long>
 			<type>bool</type>
+			<default>
+			    <value>false</value>
+			</default>
 		    </option>
 		</subgroup>
 		<subgroup>
