--- compiz-aur-patched/compizconfig/ccsm/ccm/Pages.py
+++ compiz-aur-patched/compizconfig/ccsm/ccm/Pages.py
@@ -738,42 +738,45 @@ class ProfileBackendPage(object):
         self.ProfileComboBox.handler_block (self.ProfileHandler)
 
         self.Context.Read ()
         self.Context.UpdateProfiles ()
 
         self.ProfileComboBox.get_model ().clear ()
         set = False
         for index, profile in enumerate ([_("Default")] + list (self.Context.Profiles)):
             self.ProfileComboBox.append_text (profile)
             if profile == current and not set:
                 self.ProfileComboBox.set_active (index)
                 set = True
-        self.ProfileRemoveButton.set_sensitive (self.ProfileComboBox.get_active() != 0)
+        self.ProfileRemoveButton.set_sensitive(self.ProfileComboBox.get_active() != 0)
 
         self.ProfileComboBox.handler_unblock (self.ProfileHandler)
 
         GlobalUpdater.UpdatePlugins()
 
     def IntegrationChanged(self, widget):
         value = widget.get_active()
         self.Context.Integration = value
 
     def ProfileChanged(self, widget):
         name = widget.get_active_text()
         if name == _("Default"):
-            self.Context.ResetProfile()
+            self.ResetProfile(None)
+            self.ProfileRemoveButton.set_sensitive(False)
+            GlobalUpdater.UpdatePlugins()
+            return False
         elif name in self.Context.Profiles:
             self.Context.CurrentProfile = self.Context.Profiles[name]
         else:
-            self.ProfileComboBox.set_active (0)
+            self.ProfileComboBox.set_active(0)
             return
 
         self.ProfileRemoveButton.set_sensitive (self.ProfileComboBox.get_active() != 0)
 
         self.Context.Read()
         self.Context.Write()
         GlobalUpdater.UpdatePlugins()
         return False
 
     def ProfileChangedAddTimeout(self, widget):
         GLib.timeout_add (500, self.ProfileChanged, widget)
 
@@ -788,25 +791,25 @@ class ProfileBackendPage(object):
         filter.set_name(_("All files"))
         chooser.add_filter(filter)
 
     def ResetProfile(self, widget):
         for plugin in self.Context.Plugins.values():
             settings = GetSettings(plugin)
             for setting in settings:
                 setting.Reset()
 
         activePlugins = self.Context.Plugins['core'].Screen['active_plugins'].Value
         for plugin in self.Context.Plugins.values():
             plugin.Enabled = plugin.Name in activePlugins
-        self.Context.Write()
+        self.Context.Write(onlyChanged=False)
         GlobalUpdater.UpdatePlugins()
     
     def ExportProfile(self, widget):
         main = widget.get_toplevel()
         chooser = Gtk.FileChooserDialog(title=_("Save file.."), transient_for=main, action=Gtk.FileChooserAction.SAVE)
         chooser.add_buttons(_("Cancel"), Gtk.ResponseType.CANCEL, _("Save"), Gtk.ResponseType.OK)
         chooser.set_current_folder(os.environ.get("HOME"))
         self.CreateFilter(chooser)
         ret = chooser.run()
 
         path = chooser.get_filename()
         chooser.destroy()
@@ -871,25 +874,25 @@ class ProfileBackendPage(object):
         self.Context.Import (path)
 
     def AddProfile (self, widget):
         main = widget.get_toplevel ()
         name = self.ProfileNameDialog (main)
         if name:
             self.Context.CurrentProfile = ccs.Profile (self.Context, name)
             self.UpdateProfiles (name)
     
     def RemoveProfile(self, widget):
         name = self.ProfileComboBox.get_active_text()
         if name != _("Default"):
-            self.Context.ResetProfile()
+            self.ResetProfile(None)
             self.Context.Profiles[name].Delete()
             self.UpdateProfiles()
     
     def BackendChanged(self, widget):
         shortDesc = widget.get_active_text()
         name = ""
         for backend in self.Context.Backends.values():
             if backend.ShortDesc == shortDesc:
                 name = backend.Name
                 break
         
         if name != "":
